### ISSUE TITLE
npm home gives error if there is no release version, only prereleases

### ISSUE THREAD

**Issue by aredridel** [npm/npm#8951]
```
:; npm home require2
npm ERR! Darwin 14.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "home" "require2"
npm ERR! node v0.12.7
npm ERR! npm  v3.1.2

npm ERR! No compatible version found: require2
npm ERR! Valid install targets:
npm ERR! ["0.0.0-draft.1"]
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ariastewart/Projects/file-resolver/npm-debug.log
```

Seems this could be improved to use the last entry in `versions`

**Comment by othiym23** [comment]
`npm home`, `npm repo`, and `npm bugs` all use the default `client.get` function exported by an `npm-registry-client-instance`. It _should_ use latest, regardless of the semver for that package. Actually, it should probably just use the results of fetching `https://registry.url/package-name` and not care about semver at all. I'll need to poke at `npm@2` to figure out why `npm docs` (which `npm home` is aliased to) cares about the version at all. This affects both `npm@2` and `npm@3`.

**Comment by othiym23** [comment]
Hold the bananaphone! This works fine in `npm@2` (tested against `npm@2.13.1`), so this is probably something specific to `fetchPackageMetadata` in npm@3, and I'm making this @iarna's problem. :stuck_out_tongue_winking_eye:

**Comment by iarna** [comment]
Well, I would have expected the fix to let `*` match `latest` if all releases were prereleases. Buut, that went into the cache (whence it downloads tarballs) and the whole purpose of fetch-metadata existing is that it doesn't fetch tarballs, sooo... it probably needs that logic too.

**Comment by iarna** [comment]
Released in 3.2.2
